### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Coding-with-Adam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,93 +43,94 @@ pip install dash-tools
 
 ## **Usage Examples**
 
-Below are common usage examples. For a more in-depth tutorial on writing apps for Plotly Dash, see the [Plotly Dash Documentation](https://dash.plotly.com/layout). For information about dash-tools commands, read the [_Commands_](#commands) section.
+### A. Create an App
```
Comment (andrew-hossack) [inline README.md:46]: Looks a lot cleaner now. I'm a fan of these less wordy examples.

File: README.md [hunk 2]
```diff
@@ -43,93 +43,94 @@ pip install dash-tools
...
 
-### **Creating A New Project**
-
-> _For a more in-depth tutorial on using dash-tools, check out the [Create and Upload Your App](/docs/Configuring-Your-App.md) document!_
-
-Creating a new Dash project is very simple. The following command will create a new directory called "MyDashApp":
+1. Create a Dash project in a new directory called "MyDashApp" (using your terminal or command prompt):
```
Comment (andrew-hossack) [inline README.md:48]: Should we mention here that "MyDashApp" name can be changed to anything?

File: README.md [hunk 3]
```diff
@@ -43,93 +43,94 @@ pip install dash-tools
...
 
 ```bash
 dash-tools --init MyDashApp
 ```
 
-Optionally, templates can be used. Check out [Templates](#templates) for more details.
-
-You can see what files are included with your new app:
+2. Open the default app:
```
Comment (andrew-hossack) [inline README.md:54]: What does default app mean here?

File: README.md [hunk 4]
```diff
@@ -43,93 +43,94 @@ pip install dash-tools
...
+3. Replace the code in `app.py` with your own app code. Make sure to keep the `server = app.server` right after app instantiation:
+
+![update-app](https://user-images.githubusercontent.com/32049495/169304171-bf23b2d0-26b4-4767-b38f-cd6586ddf56e.gif)
 
-When you are happy with your changes, run your dash app locally with the following command. You will be able to view your app at http://127.0.0.1:8050/ in your browser:
+4. Run your app to ensure it works:
 
 ```bash
-python src/app.py
+python .\MyDashApp\src\app.py
```
Comment (andrew-hossack) [inline README.md:67]: This command will only work on windows, as Linux and Mac OS uses forward slashes "/" to denote directories.

Should we add a second code block for Linux and Mac here that uses the forward slashes?

File: README.md [hunk 5]
```diff
@@ -43,93 +43,94 @@ pip install dash-tools
...
 ```
 
-Optionally, templates can be used. Check out [Templates](#templates) for more details.
-
-You can see what files are included with your new app:
+2. Open the default app:
 
 ```bash
-cd MyDashApp
+.\MyDashApp\src\app.py
```
Comment (andrew-hossack) [inline README.md:57]: Same comment as below on line 67, this is windows specific code

**Review by andrew-hossack — COMMENTED** [review]
Overall I'm really happy with this work! Few nit picks, but overall a definite improvement in README readability and quality.

**Comment by andrew-hossack** [comment]
Before we proceed with more marketing (YouTube video, plotly community post, medium article) I want to discuss making a core change to the CLI commands;

- Replace `dash-tools --init MyApp` with `dash-tools init MyApp`
- Replace `dash-tools --deploy-heroku` with `dash-tools heroku --deploy` (have a separate --update command for existing apps) 
- Replace `dash-tools --templates` with `dash-tools templates --list` (also add --create to turn existing directory into a template)

File: README.md [hunk 6]
```diff
@@ -43,93 +43,94 @@ pip install dash-tools
...
-See the [Templates](#templates) section for more details on using templates.
-
-### **Deploying To Heroku**
-
-Deploying your project online to [Heroku](https://www.heroku.com/) is simple. The CLI handles both creating and deploying a new app, as well as updating an existing app.
+6. Initiate git to allow deployment of app to web:
 
-#### **Using dash-tools to Create An App**
+```bash
+git init
```
[the remark this answers is not in the document]
Comment (andrew-hossack) [reply README.md:81]: Added in `dev`

File: README.md (hunk 2 above)
Comment (Coding-with-Adam) [reply README.md:48]: yes, definitely. Just added it.

File: README.md (hunk 3 above)
Comment (Coding-with-Adam) [reply README.md:54]: I changed it to "open the default app.py file that comes with this project". Feel free to tweak it even further to ensure it makes sense to you.

File: README.md (hunk 5 above)
Comment (Coding-with-Adam) [reply README.md:57]: Done

**Review by Coding-with-Adam — COMMENTED** [review]
Updated PR with Andrew's suggestions.

**Review by andrew-hossack — APPROVED** [review]

**Pull request merged**